src/core/evaluator.js

```diff
@@ -1640,19 +1650,22 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                 xObjStateManager.transform(matrix);
               }
 
+              // enqueue textContent chunk before call for sub-form.
+              enqueueChunk();
               next(self.getTextContent({
                 stream: xobj,
                 task,
                 resources: xobj.dict.get('Resources') || resources,
                 stateManager: xObjStateManager,
                 normalizeWhitespace,
                 combineTextItems,
+                sink
               }).then(function (formTextContent) {
-                Util.appendToArray(textContent.items, formTextContent.items);
-                Util.extendObj(textContent.styles, formTextContent.styles);
+                // Util.appendToArray(textContent.items, formTextContent.items);
+                // Util.extendObj(textContent.styles, formTextContent.styles);
 
-                xobjsCache.key = name;
-                xobjsCache.texts = formTextContent;
+                // xobjsCache.key = name;
+                // xobjsCache.texts = formTextContent;
```

Comment:
let sinkWrapper = {
  cachedChunks: [],
  get ready() { return sick.ready; },
  get desiredSize() { return sink.desiredSize; },
  enqueue(chunk, size) { sink.enqueue(chunk, size); this.cachedChunks.push(chunk); },
 // ... close (I don't think we are using onPull or onCancel though)